frappe/core/doctype/language/language.json

```diff
@@ -64,15 +64,6 @@
    "read": 1,
    "role": "System Manager",
    "write": 1
-  },
-  {
-   "email": 1,
-   "export": 1,
-   "print": 1,
-   "read": 1,
-   "report": 1,
-   "role": "Guest",
```

Comment:
We should add `All` at least then? 


Link fields for language exists in few places. They'll stop working all non-system managers.

Reply:
That makes sense, slipped my mind.